src/test/java/com/shippo/model/TransactionTest.java

```diff
@@ -79,7 +79,7 @@ public static Object getDefaultObject() {
 		objectMap.put("metadata", "Customer ID 123456");
 
 		try {
-			Transaction testObject = Transaction.create(objectMap);
+			Transaction testObject = Transaction.createSync(objectMap);
```

Comment:
We weren't waiting for the transaction to finish processing, so it sometimes wasn't valid to be manifested. Flakiness  cause #2